api/src/Entity/Repository/OrganisationRepository.php

```diff
@@ -12,13 +12,43 @@ class OrganisationRepository extends EntityRepository
      */
     public function getAllArray(): array
     {
+        $filter = $this->_em->getFilters()->getFilter('softdeleteable');
+        $filter->disableForEntity(Organisation::class);
+
         $query = $this
             ->getEntityManager()
             ->createQuery('SELECT o FROM App\Entity\Organisation o');
 
         return $query->getArrayResult();
     }
 
+    /**
+     * @return array
+     */
+    public function getNonDeletedArray(): array
+    {
+        $query = $this
+            ->getEntityManager()
+            ->createQuery('SELECT o FROM App\Entity\Organisation o WHERE o.deletedAt IS NULL');
```

Comment:
I have no idea if this works or not so take this comment with a pinch of salt but I wonder if this will do the same thing without the WHERE clause because of that gedmo malarkey. I think it's nice having it be more specific but not sure if the whole gedmo thing might confuse ppl in the future. Like maybe we could remove the where clause and write a comment? I don't know. To be honest it's probably fine as is. Just a thought :)